lib/mix/tasks/setup.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Mix.Tasks.LiveVue.Setup do
+  @moduledoc """
+  Copies files from assests/copy of the live_vue dependency to phoenix project assets folder
+  """
+  @shortdoc "copy setup files to assets"
+
+  use Mix.Task
+
+  @impl Mix.Task
+  # Adapted from live_svelte mix task at https://github.com/woutdp/live_svelte/blob/master/lib/mix/tasks/configure_esbuild.ex
+  def run(_args) do
+    Mix.Project.deps_paths(depth: 1)
+    |> Map.fetch!(:live_vue)
+    |> Path.join("assets/copy/**/{*.*}")
+    |> Path.wildcard(match_dot: true)
+    |> Enum.each(fn full_path ->
+      [_beginning, relative_path] = String.split(full_path, "copy", parts: 2)
+      new_path = "assets" <> relative_path
+
+      case File.exists?(new_path) do
+        true ->
+          log_info(~s/Did not copy `#{full_path}` to `#{new_path}` since file already exists/)
+
+        false ->
+          log_info(~s/Copying `#{full_path}` to `#{new_path}`/)
+          Mix.Generator.copy_file(full_path, new_path)
+          log_success(~s/Successfully copied `#{full_path}` to `#{new_path}`/)
```

Comment:
I'd say we don't need both `log_info` and `log_success` in the success case, since `copy_file` logs it as well (so we have 3 logs per success right now). 

On failure it's important because we want to point users to sources of files that were about to be copied, so they can adjust by themselves. So that's great as is 👍 

Since it's a small thing, I'll merge your PR and change it on the main branch, ok?

Reply:
Yeah, I thought it was repetitious but erred on the side of caution. It should be okay with just one output.